scripts/irods/test/test_imiscsvrinfo.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
```

Comment:
This can be dropped since we're Python 3 only.

Reply:
python3 only.... only in main.   right?

Reply:
Python 3 for 4.3 as well.

Reply:
https://docs.irods.org/4.3.3/release_notes/

> Notable updates include:
>
> - Python3 Compliance - All control scripts and the Python Rule Engine Plugin are now Python3 only. Please test any existing Python rules before upgrading.

Reply:
we did it.